projects/schematics/package.json

```diff
@@ -18,14 +18,13 @@
   "license": "MIT",
   "schematics": "./src/collection.json",
   "dependencies": {
-    "@angular-devkit/core": "^10.1.0",
-    "@angular-devkit/schematics": "^10.1.0",
-    "@angular/cdk": "^10.2.0",
     "@angular/localize": "^10.1.0",
+    "@nguniversal/express-engine": "^10.1.0"
+  },
+  "peerDependencies": {
     "@angular/pwa": "^0.1001.0",
-    "@nguniversal/express-engine": "^10.1.0",
-    "@schematics/angular": "^10.1.0",
```

Comment:
Why we removed the @schematics/angular ? I still see a lot of references to it.

Reply:
We're pulling it from the parent node_modules. I added it to the peerDeps now.